lib/http/download.go

```diff
@@ -45,21 +45,21 @@ func downloadHandler(c *fb.Context, w http.ResponseWriter, r *http.Request) (int
 
 	var (
 		extension string
-		ar        archiver.Archiver
+		ar        archiver.Writer
 	)
 
 	switch query {
 	// If the format is true, just set it to "zip".
 	case "zip", "true", "":
-		extension, ar = ".zip", archiver.Zip
+		extension, ar = ".zip", &archiver.Zip{}
```

Comment:
I recommend using `archiver.DefaultZip` (or if you use more than one, probably `archiver.NewZip()`) -- it sets some sane defaults. Probably especially important for TarGz, which has a default compression level of 0 without sane defaults :wink:

Reply:
Thank you @mholt 😄